### PR TITLE
Readds Explosive Decompression :D

### PR DESCRIPTION
**Pull request by MaltVinegar**
## About The Pull Request

Enables explosive_decompression for auxmos
Last thing to do is to fix the bug where firelocks don't engage

## Why It's Good For The Game

Explosive decompression is fun

TODO:
     auxmos.dll still needs to be updated
     Modularization

Fixes ripped from https://github.com/shiptest-ss13/Shiptest/pull/132

## Changelog
:cl:
add: Readds Monstermos
tweak: Modifies PreCompile.sh slightly
/:cl:

### CODE REVIEW

File: code/game/machinery/doors/firedoor.dm [hunk 1]
```diff
@@ -78,10 +78,19 @@
...
+		var/mob/user = AM
+		if(check_safety(user))
+			add_fingerprint(user)
+			open()
+			return TRUE
+	if(ismecha(AM))
+		var/obj/mecha/M = AM
+		if(M.occupant && check_safety(M.occupant))
+			open()
+			return TRUE
```
Comment (KubeRoot) [inline code/game/machinery/doors/firedoor.dm]: I'm sorry malt...
Missing `singulostation edit` marker

File: code/game/machinery/doors/firedoor.dm [hunk 2]
```diff
@@ -97,6 +106,23 @@
...
+			return FALSE
+
+		add_fingerprint(user)
+		if(density)
+			emergency_close_timer = world.time + RECLOSE_DELAY // prevent it from instaclosing again if in space
+			open()
+		else
+			close()
+		return TRUE
+
```
Comment (KubeRoot) [inline code/game/machinery/doors/firedoor.dm]: Missing `singulostation edit` marker

File: code/game/objects/effects/temporary_visuals/miscellaneous.dm [hunk 3]
```diff
@@ -388,6 +388,17 @@
...
+/obj/effect/temp_visual/dir_setting/space_wind
+	icon = 'icons/effects/atmospherics.dmi'
+	icon_state = "space_wind"
+	layer = FLY_LAYER
+	duration = 20
+	mouse_opacity = 0
+
+/obj/effect/temp_visual/dir_setting/space_wind/Initialize(mapload, set_dir, set_alpha = 255)
+	. = ..()
+	alpha = set_alpha
```
Comment (KubeRoot) [inline code/game/objects/effects/temporary_visuals/miscellaneous.dm]: Missing `singulostation edit` marker

File: code/modules/admin/admin_verbs.dm [hunk 4]
```diff
@@ -806,7 +806,6 @@ GLOBAL_PROTECT(admin_verbs_hideable)
 			continue
 		GM.parse_gas_string(F.initial_gas_mix)
 		F.copy_air(GM)
-		F.update_visuals()
```
Comment (KubeRoot) [inline code/modules/admin/admin_verbs.dm:810]: Missing `singulostation edit` marker
Also, do you happen to know why this is removed?

File: code/modules/atmospherics/environmental/LINDA_turf_tile.dm [hunk 5]
```diff
@@ -131,8 +131,10 @@
 
 /////////////////////////GAS OVERLAYS//////////////////////////////
 
+/turf/proc/update_visuals()
+	return
 
-/turf/open/proc/update_visuals()
+/turf/open/update_visuals()
```
Comment (KubeRoot) [inline code/modules/atmospherics/environmental/LINDA_turf_tile.dm]: Missing `singulostation edit` marker

File: code/modules/atmospherics/environmental/LINDA_turf_tile.dm [hunk 6]
```diff
@@ -191,7 +193,11 @@
 /turf/proc/process_cell(fire_count)
 
 /turf/open/proc/equalize_pressure_in_zone(cyclenum)
-/turf/open/proc/consider_firelocks(turf/T2)
+
+/turf/proc/consider_firelocks()
+	return
+
+/turf/open/consider_firelocks(turf/T2)
```
Comment (KubeRoot) [inline code/modules/atmospherics/environmental/LINDA_turf_tile.dm]: Missing `singulostation edit` marker

File: code/modules/atmospherics/environmental/LINDA_turf_tile.dm [hunk 7]
```diff
@@ -243,6 +249,8 @@
 		M = thing
 		if (!M.anchored && !M.pulledby && M.last_high_pressure_movement_air_cycle < SSair.times_fired)
 			M.experience_pressure_difference(pressure_difference * multiplier, pressure_direction, 0, pressure_specific_target)
+	if(pressure_difference > 100)
+		new /obj/effect/temp_visual/dir_setting/space_wind(src, pressure_direction, CLAMP(round(sqrt(pressure_difference) * 2), 10, 255))
```
Comment (KubeRoot) [inline code/modules/atmospherics/environmental/LINDA_turf_tile.dm]: Missing `singulostation edit` marker

File: code/controllers/subsystem/air.dm [hunk 8]
```diff
@@ -50,11 +50,11 @@ SUBSYSTEM_DEF(air)
 	var/log_explosive_decompression = TRUE // If things get spammy, admemes can turn this off.
 
 	// Max number of turfs equalization will grab.
-	var/equalize_turf_limit = 10
+	var/equalize_turf_limit = 20
```
Comment (KubeRoot) [inline code/controllers/subsystem/air.dm]: Missing `singulostation edit` marker

File: code/controllers/subsystem/air.dm [hunk 9]
```diff
@@ -50,11 +50,11 @@ SUBSYSTEM_DEF(air)
...
 	// Max number of turfs to look for a space turf, and max number of turfs that will be decompressed.
 	var/equalize_hard_turf_limit = 2000
 	// Whether equalization should be enabled at all.
-	var/equalize_enabled = FALSE
+	var/equalize_enabled = TRUE
```
Comment (KubeRoot) [inline code/controllers/subsystem/air.dm]: Missing `singulostation edit` marker

**Review by KubeRoot — DISMISSED** [review]

File: code/modules/admin/admin_verbs.dm (hunk 4 above)
Comment (MaltVinegar) [reply code/modules/admin/admin_verbs.dm:810]: Not sure, it was removed in your PR
https://github.com/SinguloStation13/SinguloStation-Blue/pull/2/files#diff-46efb60da866bae636d30b19603fa3a90545244fd0f1f5b154edaf1e11b41daf

**Comment by MaltVinegar** [comment]
When the 'Missing `singulostation edit` marker' hits

**Pull request merged**